patroni/postgresql/bootstrap.py

```diff
@@ -310,7 +310,13 @@ def create_or_update_role(self, name, password, options):
         CREATE ROLE "{0}" WITH {1};
     END IF;
 END;$$""".format(name, ' '.join(options))
-        self._postgresql.query(sql, *params)
+        self._postgresql.query('SET log_statement TO none')
+        self._postgresql.query("SET log_min_error_statement TO 'log'")
```

Comment:
If someone set's `log_min_duration` low enough or to 0, it will still log, will it not?

Reply:
Good question, we need to check it